lib/themes/dosomething/paraneue_dosomething/scss/content/_win-module.scss

```diff
@@ -1,13 +1,11 @@
 .win-module {
   .win-module__progress {
     margin-bottom: $base-spacing;
-    background-image: asset-url("images/WinBackgroundS.svg");
```

Comment:
Oh, although this one actually _does_ exist. We just don't have a loader configured for SVGs. Is the "win module" still a thing?